modules/videostab/include/opencv2/videostab/global_motion.hpp

```diff
@@ -180,6 +180,12 @@ class CV_EXPORTS ImageMotionEstimatorBase
     virtual void setMotionModel(MotionModel val) { motionModel_ = val; }
     virtual MotionModel motionModel() const { return motionModel_; }
 
+    virtual void setFrameMask(InputArray mask)
+    {
+        if (!mask.empty())
+            CV_Error(Error::StsNotImplemented, "Mask must not be empty.");
```

Comment:
> "Mask must not be empty."

"Mask support is not implemented".


This implementation is a stub for other (outside of OpenCV) implemented algorithms (they were implemented without mask).